ap_src/ap_app/static/js/teams.js

```diff
@@ -53,26 +53,47 @@ function favouriteTeam(e, user, id) {
     })
 }
 
-function LeaveTeam(e, user, redirect) {
-    var data = JSON.stringify({"username": user, "team": e.id})
-    fetch(apiURL + 'api/manage/?method=leave', {
-        method: "post",
-        body: data,
-        headers: {
-            "Content-Type": "application/json",
+/*
+Leaves team but also checks for lingering permissions and removes them
+*/
+async function LeaveTeam(e, user, token) {
```

Comment:
It looks like user is actually just a username, so this parameter name should be updated to username

Reply:
Yes I agree, this was the code that was originally in the function but an improvement could still be made as I suppose it semantically does not make much sense when reading it. The `user` argument was renamed to `username` with commit b594540